sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvaluationSuite.scala

```diff
@@ -59,6 +60,10 @@ class ExpressionEvaluationBaseSuite extends FunSuite {
 
 class ExpressionEvaluationSuite extends ExpressionEvaluationBaseSuite {
 
+  def create_row(values: Any*): Row = {
+    new GenericRow(values.map(ScalaReflection.convertToCatalyst).toArray)
```

Comment:
Yes, that's also the thing I had considered. Once the Row is accessible from user, we can not use UTF8Sting inside it. For catalyst tests, we should do the conversion manually.